### PR TITLE
HIP#32 | David | Remove deprecated endpoint and tests

### PR DESCRIPTION
**Pull request by davidviktorTW**

### CODE REVIEW

**Comment by thanakritTW** [comment]
Actually I looked at the ConsentNotificationTests since both of the controllers are quite similar, I notice that in the ConsentNotificationTests we have very few tests, so instead of removing or the tests in ConsentControllerTests is it possible to move some of the removed tests into ConsentNotificationTests

**Comment by davidviktorTW** [comment]
> Actually I looked at the ConsentNotificationControllerTests since both of the controllers are quite similar, I notice that in the ConsentNotificationTests we have very few tests, so instead of removing or the tests in ConsentControllerTests is it possible to move some of the removed tests into ConsentNotificationControllerTests

Good one! Done and added more missing test cases.

File: test/In.ProjectEKA.HipServiceTest/Consent/ConsentNotificationControllerTest.cs [hunk 1]
```diff
@@ -12,45 +13,76 @@ namespace In.ProjectEKA.HipServiceTest.Consent
...
+                    var expected = consentNotification.Notification;
+
+                    return actual.ConsentArtefactId == expected.ConsentDetail.ConsentId
+                        && actual.ConsentArtefact == expected.ConsentDetail
+                        && actual.Signature == expected.Signature
+                        && actual.Status == expected.Status
+                        && actual.ConsentManagerId == expected.ConsentId;
+                };
+
+            gatewayClient
```
Comment (thanakritTW) [inline test/In.ProjectEKA.HipServiceTest/Consent/ConsentNotificationControllerTest.cs:54]: I think you need `verify` here as well right or am I missing something

Comment (thanakritTW) [reply test/In.ProjectEKA.HipServiceTest/Consent/ConsentNotificationControllerTest.cs:54]: oh don't mind me, it is a private function not the test

**Pull request merged**